docs/ref/contrib/postgres/operations.txt

```diff
@@ -59,3 +59,34 @@ the ``django.contrib.postgres.operations`` module.
 .. class:: UnaccentExtension()
 
     Installs the ``unaccent`` extension.
+
+``Create extension using migrations``
+=====================================
+
+You can create a PostgreSQL Extension in your database through the generated
```

Comment:
I would change this portion of the sentence to "in your database using a migration file. The following"